hieradata/role/fiberspec.yaml

```diff
@@ -4,4 +4,9 @@ classes:
   - "profile::core::docker"
   - "profile::core::fiberspec"
   - "profile::core::nfsclient"
+  - "profile::core::systemd"
   - "profile::ts::dco"
+
+systemd::manage_udevd: true
```

Comment:
This is already set in `hieradata/common.yaml`.

Reply:
Moved it up on to common layer.